server/app/routes/api/wines.js

```diff
@@ -0,0 +1,102 @@
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose')
+
+router.get('/', function(req, res) {
+	mongoose.model('Wine')
+	.find({})
+	.then(function(wines) {
+		res.json(wines)
+	})
+	.then(null, function(err) {
+		console.log(err)
+	})
+})
+
+router.get('/:id', function(req, res) {
+	var id = req.params.id
+	mongoose.model('Wine')
+	.findById(id)
+	.then(function(wine) {
+		res.json(wine)
+	})
+	.then(null, function(err) {
+		console.log(err)
+		res.sendStatus(404)
+	})
+})
+
+router.get('/:id/reviews', function(req, res) {
+	var id = req.params.id
+	mongoose.model('Wine')
```

Comment:
Wine schema line 69 has the findReviews method. We changed this to be a static and updated this get request to just do one db call where we are doing Wine.findReviews(id).

However, should this be moved to the reviews route, as the reviews route should be handling both reviews associated with a user and reviews associated with a wine?